modules/aws/ami/README.md

```diff
@@ -1,27 +1,40 @@
 # Container Linux AMI Module
 
 This [Terraform][] [module][] supports `latest` versions for [Container Linux][container-linux] release channels and returns an appropriate [AMI][].
+It uses [implicit provider inheritance][implicit-provider-inheritance] to access the [AWS provider][AWS-provider].
 
 ## Example
 
-From the module directory:
+Set up a `main.tf` with:
+
+```hcl
+provider "aws" {
+  region = "us-east-1"
+}
+
+module "ami" {
+  source = "github.com/openshift/installer//modules/aws/ami"
```

Comment:
`//` is that necessary?

Reply:
> // is that necessary?

Yup.  Doc links in the commit message (b68ac26f4da8).